compiler/src/iree/compiler/ConstEval/test/jit_globals.mlir

```diff
@@ -22,7 +22,7 @@ module @linalg_tensor_jit {
     return %hoisted : tensor<5x6xf32>
   }
   // CHECK-NOT: util.initializer
-  util.initializer {
+  util.initializer attributes {iree.compiler.consteval} {
```

Comment:
I may try to add a `util.initializer.constexpr` that is an initializer but with verification that once constexpr it has to stay constexpr all the way to the end - this would help catch errors where we insert non-const-state during lowerings. Probably fiddly to get right, though, so this is fine for now.

Reply:
Yeah, that might be a good cleanup. Probably is fiddly.